### PR TITLE
Use explicit arguments for address truncation

### LINKED ISSUE #335: Change the way the connected address is shown
It should be `0x<4 chars>...<4chars>`

### PR DESCRIPTION
**Pull request by hpmaxi**
Closes #335

### CODE REVIEW

File: app/src/components/common/network/index.tsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ export const Network: React.FC = props => {
       <ConnectionStatusWrapper>
         <ConnectionStatusDot />
         <ConnectionStatusText>
-          {truncateStringInTheMiddle(account) || 'No account connected'}
+          {truncateStringInTheMiddle(account, 41, 6, 4) || 'No account connected'}
```
Comment (fvictorio) [inline app/src/components/common/network/index.tsx]: I don't remember the reason for the second parameter. I think just indicating the length of the first part and the second part should be enough: then if the length of the string is less than or equal to the sum of both arguments, you just return it.

For example:

```javascript
truncate("foobarbaz", 3, 2) // "foo...az"
truncate("foobar", 4, 2) // "foobar"
```

@mariano-aguero do you agree?

@hpmaxi can you do this (and also add tests for it)?

Comment (mariano-aguero) [reply app/src/components/common/network/index.tsx]: Yes, I agree, we don't need the second parameter

Comment (hpmaxi) [reply app/src/components/common/network/index.tsx]: I hope there's enough tests

File: app/src/util/tools.spec.ts [hunk 2]
```diff
@@ -177,4 +178,18 @@ describe('tools', () => {
       ).toThrow()
     })
   })
+
+  describe('truncateStringInTheMiddle', () => {
+    it('should not change string with truncate positions great or equal than string length', () => {
+      expect(truncate('foobar', 4, 2)).toBe('foobar')
+      expect(truncate('foobar', 6, 6)).toBe('foobar')
+      expect(truncate('foo')).toBe('foo')
```
Comment (fvictorio) [inline app/src/util/tools.spec.ts]: I don't know if it makes sense to have default arguments for this method.

Comment (mariano-aguero) [reply app/src/util/tools.spec.ts]: > I don't know if it makes sense to have default arguments for this method.

Maybe, the last two arguments must be required ?

Comment (hpmaxi) [reply app/src/util/tools.spec.ts]: I've removed the last two parameters default values to make them explicit on each call of the function

**Pull request merged**